Cargo.toml

```diff
@@ -56,11 +59,6 @@ exclude = [
   "tests/rustdoc-gui",
   # HACK(eddyb) This hardcodes the fact that our CI uses `/checkout/obj`.
   "obj",
-  # The `x` binary is a thin wrapper that calls `x.py`, which initializes
-  # submodules, before which workspace members cannot be invoked because
-  # not all `Cargo.toml` files are available, so we exclude the `x` binary,
-  # so it can be invoked before the current checkout is set up.
-  "src/tools/x",
```

Comment:
please also update `linkedProjects` in `src/etc/rust_analyzer_settings.json`.

Reply:
it's kinda annoying that there are four different files to update now. i wish we would instead autogenerate them from a canonical version.

Reply:
Done!